docker-storage-setup.1

```diff
@@ -31,14 +31,11 @@ functionality to work properly.
 \f[B]Supported options for the configuration file\f[]:
 
 STORAGE_DRIVER:
-      Specify a storage driver to be used with docker. Default
-      driver is "devicemapper". Other valid values are overlay,
-      overlay2 and "". Empty string tells docker-storage-setup to
-      not do any storage setup.
+      Specify a storage driver to be used with container runtime. The default driver is "devicemapper". Valid values are overlay, overlay2 and "". Empty string tells docker-storage-setup to not perform any storage setup.
 
-EXTRA_DOCKER_STORAGE_OPTIONS:
-      A set of extra options that should be passed to the Docker
-      daemon.
+EXTRA_STORAGE_OPTIONS:
+      A set of extra options that should be passed to the container runtime daemon.
+      NOTE: This option replaces EXTRA_DOCKER_STORAGE_OPTIONS which has been deprecated
```

Comment:
Sure I just wanted to get agreement on the man page and config then I can add them back to the other pull request.  I just saw this as a lot of noice and wanted to take care of these first.